bbreadfront/src/ProfileForm.js

```diff
@@ -22,7 +22,11 @@ function ProfileForm(props) {
   }
 
   return (
-      null
+    <div><input type="button" value="Login" onClick={props.addFriend} />
```

Comment:
Maybe think about changing the value here to a more descriptive name than Login, as we are not logging in.

Reply:
agreed, as far as the merge conflict goes for this part of the code, I think the <div> tag containing the buttons can just be added alongside the button that redirects to the new grocery run page. Not a big conflict